Examples/Image/Detection/utils/configs/VGG16_config.py

```diff
@@ -23,7 +23,9 @@
 __C.MODEL.FEATURE_STRIDE = 16
 __C.MODEL.RPN_NUM_CHANNELS = 512
 __C.MODEL.ROI_DIM = 7
-__C.MODEL.E2E_LR_FACTOR = 1.0
-__C.MODEL.RPN_LR_FACTOR = 1.0
+## In using Faster R-CNN, try E2E_LR_FACTOR or RPN_LR_FACTOR from 0.1 or smaller.
+## They decide learning rate in DNN.
```

Comment:
"They decide learning rate in DNN." - Is this comment for the user? If it is not absolutely necessary, please remove the comment.